pkg/plugins/manager/pipeline/bootstrap/factory.go

```diff
@@ -74,6 +76,13 @@ func (f *DefaultPluginFactory) newPlugin(p plugins.FoundPlugin, class plugins.Cl
 	if err = setImages(plugin, f.assetPath, info); err != nil {
 		return nil, err
 	}
+
+	if f.cfg.Features.LocalizationForPlugins {
```

Comment:
nit: If you only need the `Features` info, better to pass down only that and not the whole `cfg`.

Reply:
Fixed 👍